openidl-common-lib/middlewares/DBService/DBManagerFactory.js

```diff
@@ -1,39 +1,35 @@
 const log4js = require('log4js');
 const CloudantDBManager = require('./cloudantDBManager');
 const MongoDBManager = require('./mongoDBManager');
+const PostgresDBManager = require('./postgresDBManager');
+
 const logger = log4js.getLogger('DBManagerFactory');
 logger.level = process.env.LOG_LEVEL || 'debug';
 
 class DBManagerFactory {
-    constructor() {
+  async getInstance(options, dbType) {
+    logger.info(`Inside DBManagerFactory getInstance(${dbType}): ${JSON.stringify(options)}`);
+    let databaseType = dbType;
```

Comment:
You can write the databaseType definition and assignment in 1 line:
`let databaseType = dbType || options.defaultDbType`